examples/Cargo.toml

```diff
@@ -29,6 +29,9 @@ optional = true
 version = "0.6"
 optional = true
 
+[build-dependencies.gtk]
+path = "../gtk"
```

Comment:
Shouldn't it be gio then? It seems a bit overkill to import the whole `gtk` crate if you only use `gio`.